README.md

```diff
@@ -1,36 +1,39 @@
-# lit-html 2.0 monorepo
+# Lit 2.0 Monorepo
```

Comment:
I think we should prep for `lit-next` being moved to `main` and skip the "2.0" and calling it "development monorepo" and such.

This update looks good though, we can do another pass for that.

Reply:
👍  Went ahead and made those changes.